specifications/dpe-irot-profile/README.md

```diff
@@ -0,0 +1,16 @@
+# DPE iRoT Profile
+
+## Rendering
+
+The DPE iRoT profile is written and maintained in markdown, however the
```

Comment:
Oh dang, I forgot to update this with the OCP rendering instructions. I'll do that.